tools-local/Microsoft.ML.AutoML.SourceGenerator/Microsoft.ML.AutoML.SourceGenerator.csproj

```diff
@@ -0,0 +1,43 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <LangVersion>9.0</LangVersion>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GetTargetPathDependsOn>$(GetTargetPathDependsOn);GetDependencyTargetPaths</GetTargetPathDependsOn>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.9.0" PrivateAssets="all" />
```

Comment:
Can these versions come from versions.props?

Reply:
The version in `Versions.props` is too old and it doesn't support source generator, if we can update the version of code analysis package we can probably use that version

Reply:
Would just updating it to this version here be enough? I can test that out.

Reply:
Probably, do you want me to launch another pr to fix it?

Reply:
If you want to create a new pr with the version changes go for it. If not I can.

Reply:
#6093

Reply:
### Resolved